### PR TITLE
adding sandnes_no

### PR DESCRIPTION
**Pull request by 5ila5**
requested by #1190

tests:
```
Testing source sandnes_no ...
  found 22 entries for TestcaseI
    2023-09-05 : Papir [mdi:recycle]
    2023-09-05 : Restavfall [mdi:trash-can]
    2023-09-12 : Våtorganisk avfall [mdi:leaf]
    2023-09-19 : Restavfall [mdi:trash-can]
    2023-09-26 : Våtorganisk avfall [mdi:leaf]
    2023-10-03 : Papir [mdi:recycle]
    2023-10-03 : Restavfall [mdi:trash-can]
    2023-10-10 : Våtorganisk avfall [mdi:leaf]
    2023-10-17 : Restavfall [mdi:trash-can]
    2023-10-24 : Våtorganisk avfall [mdi:leaf]
    2023-10-31 : Papir [mdi:recycle]
    2023-10-31 : Restavfall [mdi:trash-can]
    2023-11-07 : Våtorganisk avfall [mdi:leaf]
    2023-11-14 : Restavfall [mdi:trash-can]
    2023-11-21 : Våtorganisk avfall [mdi:leaf]
    2023-11-28 : Papir [mdi:recycle]
    2023-11-28 : Restavfall [mdi:trash-can]
    2023-12-05 : Våtorganisk avfall [mdi:leaf]
    2023-12-12 : Restavfall [mdi:trash-can]
    2023-12-19 : Våtorganisk avfall [mdi:leaf]
    2023-12-27 : Papir [mdi:recycle]
    2023-12-27 : Restavfall [mdi:trash-can]
  found 21 entries for TestcaseII
    2023-09-04 : Våtorganisk avfall [mdi:leaf]
    2023-09-11 : Restavfall [mdi:trash-can]
    2023-09-18 : Våtorganisk avfall [mdi:leaf]
    2023-09-18 : Papir [mdi:recycle]
    2023-09-25 : Restavfall [mdi:trash-can]
    2023-10-02 : Våtorganisk avfall [mdi:leaf]
    2023-10-09 : Restavfall [mdi:trash-can]
    2023-10-16 : Våtorganisk avfall [mdi:leaf]
    2023-10-16 : Papir [mdi:recycle]
    2023-10-23 : Restavfall [mdi:trash-can]
    2023-10-30 : Våtorganisk avfall [mdi:leaf]
    2023-11-06 : Restavfall [mdi:trash-can]
    2023-11-13 : Våtorganisk avfall [mdi:leaf]
    2023-11-13 : Papir [mdi:recycle]
    2023-11-20 : Restavfall [mdi:trash-can]
    2023-11-27 : Våtorganisk avfall [mdi:leaf]
    2023-12-04 : Restavfall [mdi:trash-can]
    2023-12-11 : Våtorganisk avfall [mdi:leaf]
    2023-12-11 : Papir [mdi:recycle]
    2023-12-18 : Restavfall [mdi:trash-can]
    2023-12-27 : Våtorganisk avfall [mdi:leaf]
```

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/sandnes_no.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# nearly identical to stavanger_no
+
+from datetime import datetime
+
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Stavanger Kommune"
```
Comment (sjabby) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/sandnes_no.py:9]: Should say Sandnes Kommune

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/sandnes_no.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+DESCRIPTION = "Source for Stavanger Kommune, Norway"
```
Comment (sjabby) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/sandnes_no.py:10]: Source for Sandnes Kommune, Norway

File: doc/source/sandnes.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        id: "181e5aac-3c88-4b0b-ad46-3bd246c2be2c"
+        municipality: "Sandnes kommune 2020"
+        gnumber: "62"
+        bnumber: "281"
+        snumber: "0"
+```
+
+## How to get the source arguments
+
+Visit the [Stavanger Kommune, Norway](https://www.hentavfall.no/rogaland/sandnes/tommekalender) page and search for your address.
```
Comment (sjabby) [inline doc/source/sandnes.md:52]: Visit the [Sandnes Kommune, Norway](https://www.hentavfall.no/rogaland/sandnes/tommekalender) page and search for your address.

File: info.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ Waste collection schedules from service provider web sites are updated daily, de
 | Luxembourg | Esch-sur-Alzette |
 | Netherlands | ACV Group, Alpen an den Rijn, Area Afval, Avalex, Avri, Bar Afvalbeheer, Circulus, Cyclus NV, Dar, Den Haag, GAD, Gemeente Almere, Gemeente Berkelland, Gemeente Cranendonck, Gemeente Hellendoorn, Gemeente Lingewaard, Gemeente Meppel, Gemeente Middelburg + Vlissingen, Gemeente Peel en Maas, Gemeente Schouwen-Duiveland, Gemeente Sudwest-Fryslan, Gemeente Venray, Gemeente Voorschoten, Gemeente Waalre, Gemeente Westland, HVC Groep, Meerlanden, Mijn Blink, PreZero, Purmerend, RAD BV, Reinis, Spaarnelanden, Twente Milieu, Waardlanden, Ximmio, ZRD |
 | New Zealand | Auckland Council, Christchurch City Council, Dunedin District Council, Gore, Invercargill & Southland, Hamilton City Council, Horowhenua District Council, Waipa District Council, Wellington City Council |
-| Norway | Min Renovasjon, Movar IKS, Oslo Kommune, ReMidt Orkland muni, Stavanger Kommune, Trondheim |
+| Norway | Min Renovasjon, Movar IKS, Oslo Kommune, ReMidt Orkland muni, Stavanger Kommune, Stavanger Kommune, Trondheim |
```
Comment (sjabby) [inline info.md:29]: | Norway | Min Renovasjon, Movar IKS, Oslo Kommune, ReMidt Orkland muni, Stavanger Kommune, Sandnes Kommune, Trondheim |

File: README.md [hunk 5]
```diff
@@ -642,6 +642,7 @@ Waste collection schedules in the following formats and countries are supported.
 - [Movar IKS](/doc/source/movar_no.md) / movar.no
 - [Oslo Kommune](/doc/source/oslokommune_no.md) / oslo.kommune.no
 - [ReMidt Orkland muni](/doc/source/remidt_no.md) / remidt.no
+- [Stavanger Kommune](/doc/source/sandnes_no.md) / sandnes.kommune.no
```
Comment (sjabby) [inline README.md:645]: [Sandnes Kommune](/doc/source/sandnes_no.md) / sandnes.kommune.no

Comment (5ila5) [reply README.md:645]: Fixes itself when changing the `TITLE` in the py file (and running update_doculinks.py)

File: info.md (hunk 4 above)
Comment (5ila5) [reply info.md:29]: Fixes itself when changing the `TITLE` in the py file (and running update_doculinks.py)

File: doc/source/sandnes.md (hunk 3 above)
Comment (5ila5) [reply doc/source/sandnes.md:52]: fixed by #1206

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/sandnes_no.py (hunk 2 above)
Comment (5ila5) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/sandnes_no.py:10]: fixed by #1206

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/sandnes_no.py (hunk 1 above)
Comment (5ila5) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/sandnes_no.py:9]: fixed by #1206

**Pull request merged**